### PR TITLE
[optimize](invert index) optimize inverted index metadata into file cache

### PR DESCRIPTION
**Pull request by zzzxl1993**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by zzzxl1993** [comment]
run buildall

File: be/src/olap/rowset/beta_rowset.cpp [hunk 1]
```diff
@@ -347,11 +351,52 @@ Status BetaRowset::upload_to(io::RemoteFileSystem* dest_fs, const RowsetId& new_
...
+                        } catch (CLuceneError& err) {
+                            LOG(WARNING) << "opening index reader clucene err: " << err.what();
+                        } catch (...) {
+                            LOG(WARNING) << "opening index reader other err";
+                        }
+                    }
+                }
+                auto duration =
+                        std::chrono::duration<float>(std::chrono::steady_clock::now() - start);
+                LOG(INFO) << "cooldown upload open invert index duration: " << duration.count();
```
Comment (xiaokang) [inline be/src/olap/rowset/beta_rowset.cpp:396]: DEBUG

**Review by xiaokang — DISMISSED** [review]
LGTM

**Comment by xiaokang** [comment]
@platoneko can you have a look

**Comment by zzzxl1993** [comment]
run buildall

**Comment by zzzxl1993** [comment]
run buildall

**Comment by zzzxl1993** [comment]
run buildall

**Comment by zzzxl1993** [comment]
run buildall

**Pull request closed without merging**